apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -93,9 +96,9 @@ suite('thread_list_ui', function() {
         '<li id="thread-' + i + '" class="threadlist-item" ' +
             'data-time="' + someDate + '" ' +
             'data-thread-id="' + i + '">' +
-          '<label>' +
-            '<input type="checkbox" data-mode="threads" value="' + i + '"/>' +
-          '</label>' +
+          '<gaia-checkbox data-mode="threads" value="' + i + '">' +
```

Comment:
just wondering why you don't use inheritance like `<input type='checkbox' use='gaia-checkbox'>` ?